docs/concepts/models.md

```diff
@@ -524,182 +500,126 @@ except ValidationError as e:
     """
 ```
 
-If you want to validate serialized data in a format other than JSON, you should load the data into a dict yourself and
+If you want to validate serialized data in a format other than JSON, you should load the data into a dictionary yourself and
 then pass it to [`model_validate`][pydantic.main.BaseModel.model_validate].
 
 !!! note
     Depending on the types and model configs involved, [`model_validate`][pydantic.main.BaseModel.model_validate]
     and [`model_validate_json`][pydantic.main.BaseModel.model_validate_json] may have different validation behavior.
     If you have data coming from a non-JSON source, but want the same validation
     behavior and errors you'd get from [`model_validate_json`][pydantic.main.BaseModel.model_validate_json],
-    our recommendation for now is to use either use `model_validate_json(json.dumps(data))`, or use [`model_validate_strings`][pydantic.main.BaseModel.model_validate_strings] if the data takes the form of a (potentially nested) dict with string keys and values.
-
-!!! note
-    Learn more about JSON parsing in the [JSON](../concepts/json.md) section of the docs.
+    our recommendation for now is to use either use `model_validate_json(json.dumps(data))`, or use [`model_validate_strings`][pydantic.main.BaseModel.model_validate_strings] if the data takes the form of a (potentially nested) dictionary with string keys and values.
 
 !!! note
     If you're passing in an instance of a model to [`model_validate`][pydantic.main.BaseModel.model_validate], you will want to consider setting
-    [`revalidate_instances`](https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.revalidate_instances)
-    in the model's config. If you don't set this value, then validation will be skipped on model instances. See the below example:
+    [`revalidate_instances`][pydantic.ConfigDict.revalidate_instances] in the model's config.
+    If you don't set this value, then validation will be skipped on model instances. See the below example:
 
+    === ":x: `revalidate_instances='never'`"
```

Comment:
Example was indented correctly, almost nothing changed (some rewording in comments).